arc/cli/commands/diff.py

```
@@ -400,16 +406,44 @@ def _perform_statistical_analysis(results_a: list, results_b: list) -> Dict[str,
     p_b = sum(successes_b) / n_b if n_b > 0 else 0
     
     # Use chi-square test for proportions (more appropriate than t-test for binary data)
-    from scipy.stats import chi2_contingency
+    from scipy.stats import chi2_contingency, fisher_exact
```

Ruff (0.11.9)

Redefinition of unused `chi2_contingency` from line 12

Remove definition: `chi2_contingency`

(F811)